Specs/Scene/ShadowMapSpec.js

```diff
@@ -797,7 +827,17 @@ describe(
       scene.light = new DirectionalLight({
         direction: lightDirectionAngle,
       });
-      renderAndExpect(unshadowedColor);
+
+      // Change the time so that the shadows are no longer pointing straight down
+      renderAndCall(function (rgba) {
+        // Due to shading differences, the result might not exactly the same, but
```

Comment:
What does "shading differences" refer to here -- differences between new `Model` and old `Model`?